### PR TITLE
[Fix] Fix TensorRT 7 SSD

### PR DESCRIPTION
**Pull request by grimoire**
TensorRT 7 does not support dynamic clamp, which is used in `torch.nn.functional.normalize`.

### CODE REVIEW

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by SingleZombie — APPROVED** [review]
LGTM
